.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -12,6 +9,7 @@ A clear and concise description of what the bug is.
 
 **To Reproduce**
 Steps to reproduce the behavior:
+
```

Comment:
Can we remove this extra line please? Or is this intentional?

Reply:
It's intentional, it's made by the formatter.